src/v1_0/rw/elog.cpp

```diff
@@ -132,11 +146,14 @@ namespace abb ::rws ::v1_0 ::rw ::elog
         {
             std::stringstream arg_name;
             arg_name << "arg" << i;
-            auto arg_node = xmlFindNodes(node, XMLAttribute("class", arg_name.str()))[0];
-
-            std::string value = arg_node->innerText();
-            std::string valueType = xmlNodeGetAttributeValue(arg_node, "type");
-
+            auto arg_nodes = xmlFindNodes(node, XMLAttribute("class", arg_name.str()));
+            std::string value = "STRING";
+            std::string valueType = "UNDEFIEND";
```

Comment:
:+1:  Here I would keep "UNDEFINED", since it makes more sense than empty string in this context